platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -54,20 +54,72 @@ For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility
+Relevant problems in the report are now rendered in the console output when you use `--warning-mode=all`:
+
+```bash
+$ ./gradlew :test --warning-mode=all
+
+> Configure project :
+Build file '/Users/lkassovic/Downloads/test-report-aggregation-sample/build.gradle': line 16
+The Wrapper.getAvailableDistributionTypes method has been deprecated...
+        at build_5teuix0v7qf7ou93kgnmvnicp.run(/Users/user/test-report-aggregation-sample/build.gradle:16)
+        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
+[Incubating] Problems report is available at: file:///Users/user/test-report-aggregation-sample/build/reports/problems/problems-report.html
+```
+
+#### Clearer explanation for worker process exit codes
+
+Gradle now provides a short explanation when a [worker process](userguide/gradle_daemon.html#understanding_daemon) exits with a code that typically indicates it was killed by the operating system.
+
+Previously, you would see:
+
+```bash
+> Process 'Gradle Worker Daemon 87' finished with non-zero exit value 137
+```
+
+Now, Gradle adds a helpful hint:
+
+```bash
+> Process 'Gradle Worker Daemon 87' finished with non-zero exit value 137 
+(this value may indicate that the process was terminated with the SIGKILL 
+signal, which is often caused by the system running out of memory)
+```
 
-Gradle provides APIs for testing build logic and plugins with [TestKit](userguide/test_kit.html), and a dedicated [`Test` task](userguide/java_testing.html) for JVM code.
+This makes it easier to understand failures caused by conditions such as running out of memory.
 
-#### Testing refactoring
+### Testing reporting improvements
 
-The [HTML Test Report](userguide/java_testing.html#test_reporting) generated by the `test` task, `TestReport`, and other `AbstractTestTask` users now provides a more detailed view of the test results.
-This is due to a refactoring of the testing infrastructure to use the incubating `TestEventReporter` API internally.
-Additionally, the binary output has been adjusted to better support non-class-based testing frameworks.
+Gradle provides a [rich set of features and abstractions](userguide/java_testing.html) for testing JVM code, along with test reports to display results.
 
-In the following sections:
+#### Test results reporting
 
-- "Suite" refers specifically to the suite features of JUnit 4, JUnit Jupiter, and TestNG.
-- "Container" refers to a general grouping of tests, such as a class.
+The [HTML test report](userguide/java_testing.html#test_reporting) generated by the `test` task, [`TestReport`](javadoc/org/gradle/api/tasks/testing/TestReport.html), and other [`AbstractTestTask`](javadoc/org/gradle/api/tasks/testing/AbstractTestTask.html) usages now presents test results in a clearer structure that reflects how tests are defined:
+
+![test-results-reporting-packages.png](release-notes-assets/test-results-reporting-packages.png)
```

Comment:
I am confused by this, this is the old report, not the new one. This is _gone_ now.